### PR TITLE
Bug 1536285 - Add Highlights as the final discovery stream section

### PR DESCRIPTION
**Pull request by Mardak**
r?@ScottDowne Wraps similar to what we did for TopSites and adjusting the styling to fit with discoverystream cards:

Screen Shot 2019-06-14 at 5 32 50 PM

Screen Shot 2019-06-14 at 2 29 28 PM

### CODE REVIEW

**Comment by wolasi** [comment]
@Mardak  Any chance we can make some of the minor adjust outlined in the spec e.g. making the source Title case instead of uppercase? That alone would be great for consistency.

**Comment by Mardak** [comment]
Updated some styling from https://www.figma.com/file/eGdZVYlZ5evvePHUCtKnYg7u/New-Tab-Spec-for-Fx-70 (edited images in original comment)

**Comment by wolasi** [comment]
@Mardak  can we make the following changes?
- even up the top and bottom padding on the text block
- make the font-size of the bold text 14px with line-height 20px

**Comment by Mardak** [comment]
Are you saying the screenshot doesn't have those sizes or that's what they should be?

File: lib/ActivityStream.jsm [hunk 1]
```diff
@@ -145,7 +145,7 @@ const PREFS_CONFIG = new Map([
   }],
   ["section.highlights.rows", {
     title: "Number of rows of Highlights to display",
-    value: 2,
+    value: 1,
```
Comment (punamdahiya) [inline lib/ActivityStream.jsm]: why this change?

Comment (Mardak) [reply lib/ActivityStream.jsm]: for a single row:
Screen Shot 2019-06-14 at 7 04 51 PM

**Comment by Mardak** [comment]
Filed download dismissing https://bugzilla.mozilla.org/show_bug.cgi?id=1559548
and new bookmarks https://bugzilla.mozilla.org/show_bug.cgi?id=1559552
